homeassistant/components/watson_tts/tts.py

```diff
@@ -0,0 +1,136 @@
+"""Support for IBM Watson TTS integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.tts import PLATFORM_SCHEMA, Provider
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_URL = 'watson_url'
+CONF_APIKEY = 'watson_apikey'
+ATTR_CREDENTIALS = 'credentials'
+
+DEFAULT_URL = 'https://stream.watsonplatform.net/text-to-speech/api'
+
+CONF_VOICE = 'voice'
+CONF_OUTPUT_FORMAT = 'output_format'
+CONF_TEXT_TYPE = 'text'
+
+SUPPORTED_VOICES = [
```

Comment:
Where can we update this list?

Reply:
what do you mean?

Reply:
I added the URL which points to the relevant docs page